clients/src/main/java/org/apache/kafka/common/requests/JoinGroupResponse.java

```diff
@@ -30,63 +31,79 @@
     /**
      * Possible error code:
      *
-     * CONSUMER_COORDINATOR_NOT_AVAILABLE (15)
-     * NOT_COORDINATOR_FOR_CONSUMER (16)
-     * INCONSISTENT_PARTITION_ASSIGNMENT_STRATEGY (23)
+     * GROUP_COORDINATOR_NOT_AVAILABLE (15)
+     * NOT_COORDINATOR_FOR_GROUP (16)
+     * INCONSISTENT_GROUP_PROTOCOL (23)
      * UNKNOWN_PARTITION_ASSIGNMENT_STRATEGY (24)
```

Comment:
Is this still a possible error code? We may need to update the possible error codes for Join / Sync / Commit / FetchOffset responses.